### PR TITLE
Ci 649 invest contact form

### PR DESCRIPTION
**Pull request by jeanettemariemurphy**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -18,100 +18,124 @@ msgstr ""
...
 msgstr "Unternehmenswebseite"
 
-#: contact/forms.py:52 core/forms.py:105
-msgid "Which country are you based in?"
-msgstr "Hauptsitz ihres Unternehmens"
+#: contact/forms.py:78 contact/forms.py:138
+#, fuzzy
+#| msgid "Company name"
+msgid "Company HQ address"
+msgstr "Eingetragener Firmenname"
```
Comment (eadpearce) [inline locale/de/LC_MESSAGES/django.po]: this translation is now incorrect

Comment (eadpearce) [reply locale/de/LC_MESSAGES/django.po]: it should be "Hauptsitz ihres Unternehmens"

File: locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -202,18 +226,19 @@ msgstr ""
...
+msgid "Email address"
+msgstr "Arbeits-E-Mail"
 
 #: core/forms.py:101
 msgid "Phone number (Optional)"
 msgstr "Arbeits-Telefon"
 
+#: core/forms.py:105
+msgid "Which country are you based in?"
+msgstr "Hauptsitz ihres Unternehmens"
```
Comment (eadpearce) [inline locale/de/LC_MESSAGES/django.po]: this one is also wrong. just leave it blank

File: contact/forms.py [hunk 3]
```diff
@@ -1,75 +1,111 @@
...
 
 from django.conf import settings
 from django.forms import Textarea, Select
 from django.template.loader import render_to_string
 from django.utils.translation import ugettext_lazy as _
 
 
-COUNTRIES = [(label, label) for _, label in choices.COUNTRY_CHOICES]
-
+COUNTRIES = BLANK_CHOICE_DASH + list(COUNTRY_CHOICES)
```
Comment (eadpearce) [inline contact/forms.py]: you don't need to do `list()` on `COUNTRY_CHOICES` as it's already a list

File: contact/forms.py [hunk 4]
```diff
@@ -92,17 +128,23 @@ def get_context_data(self):
...
-                (_('Your investment'), data['description']),
+                (_('Industry'), data['industry']),
+                (_('Which of these best describes how you feel about expanding to the UK?'), data['expanding_to_uk']),
+                (_('Tell us about your investment'), data['description']),
+                (_('Would you like us to arrange a call?'), data['arrange_callback']),
                 (_('I would like to be contacted by email'), data['email_contact_consent']),
-                (_('I would like to be contacted by telephone'), data['telephone_contact_consent'])
+                (_('I would like to be contacted by telephone'), data['telephone_contact_consent']),
+
+                # (_('When should we call you?'), data['when_to_call'])
```
Comment (eadpearce) [inline contact/forms.py]: remove comment

File: contact/tests/test_forms.py [hunk 5]
```diff
@@ -134,7 +150,7 @@ def test_render_agent_email_context(contact_form_data):
         utm_data={'field_one': 'value_one'},
         submission_url='http://www.google.com/submission_url'
     )
-
+    # import pdb; pdb.set_trace()
```
Comment (eadpearce) [inline contact/tests/test_forms.py]: remove comment

File: contact/forms.py (hunk 3 above)
Comment (jeanettemariemurphy) [reply contact/forms.py]: fixed

File: contact/forms.py (hunk 4 above)
Comment (jeanettemariemurphy) [reply contact/forms.py]: fixed

File: contact/tests/test_forms.py (hunk 5 above)
Comment (jeanettemariemurphy) [reply contact/tests/test_forms.py]: fixed

File: locale/de/LC_MESSAGES/django.po (hunk 1 above)
Comment (jeanettemariemurphy) [reply locale/de/LC_MESSAGES/django.po]: fixed

**Pull request merged**